### PR TITLE
Run compute_ctl in a cgroup in VMs

### PR DESCRIPTION
**Pull request by sharnoff**
Not expected to work without <https://github.com/neondatabase/neonvm/pull/41> and <https://github.com/neondatabase/autoscaling/pull/40>.

Related work: <https://github.com/neondatabase/autoscaling/pull/41>.

---

I know this _mostly_ looks like unrelated changes. The basic idea is: It's easier to manage cgroups and the vm-informant outside of compute_ctl. With a little bit of added functionality to NeonVM, it's possible to implement this pretty simply, while keeping compute_ctl running as user `postgres` (instead of e.g., root, to manage cgroups).

~~This PR is mostly intended to be exploratory - as an alternative to writing a custom vm-builder for this repo. IMO this approach will be simpler and easier to maintain.~~

_edit:_ We decided to go with this approach for now.

---

The first two commits are from the previous approach I used.

### CODE REVIEW

File: Dockerfile.vm-compute-node [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Pull VM informant and set up inittab
+FROM neondatabase/vm-informant:$VM_INFORMANT_VERSION as informant
+
+RUN set -e \
+	&& rm -f /etc/inittab \
+	&& touch /etc/inittab
+
+ADD vm-cgconfig.conf /etc/cgconfig.conf
+RUN echo "::sysinit:cgconfigparser -l /etc/cgconfig.conf -s 1664" >> /etc/inittab
+RUN echo "::respawn:su vm-informant -c '/bin/vm-informant --auto-restart --cgroup=neon-postgres"
```
Comment (tychoish) [inline Dockerfile.vm-compute-node:15]: fix the thing

**Review by tychoish — APPROVED** [review]

File: Dockerfile.vm-compute-node (hunk 1 above)
Comment (sharnoff) [reply Dockerfile.vm-compute-node]: thing fixed :)

File: Dockerfile.vm-compute-node [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Pull VM informant and set up inittab
+FROM neondatabase/vm-informant:$VM_INFORMANT_VERSION as informant
+
+RUN set -e \
+	&& rm -f /etc/inittab \
+	&& touch /etc/inittab
+
+ADD vm-cgconfig.conf /etc/cgconfig.conf
+RUN echo "::sysinit:cgconfigparser -l /etc/cgconfig.conf -s 1664" >> /etc/inittab
+RUN echo "::respawn:su vm-informant -c '/bin/vm-informant --auto-restart --cgroup=neon-postgres" >> /etc/inittab
```
Comment (vadim2404) [inline Dockerfile.vm-compute-node:15]: Do you need two different layers?

Comment (sharnoff) [reply Dockerfile.vm-compute-node]: Nope. Merged them

**Comment by sharnoff** [comment]
Merging this, will monitor staging to check it works ok.

**Pull request merged**